platforms/software/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationSpec.groovy

```diff
@@ -799,7 +800,7 @@ This method is only meant to be called on configurations which allow the (non-de
             assert copy.attributes.getAttribute(it) == original.attributes.getAttribute(it)
         }
         assert copy.canBeResolved == original.canBeResolved
-        assert copy.canBeConsumed == original.canBeConsumed
+        assert !copy.canBeConsumed // Copies are now made as RESOLVABLE_DEPENDENCY_SCOPE, so are non-consumable regardless of the usage of the original
```

Comment:
I would just change these lines to 

```
assert copy.canBeResolved
assert !copy.canBeConsumed
```

`copy.canBeResolved == original.canBeResolved` is a very odd check to make, since you are not allowed to copy a non-resolvable configuration. That means `original.canBeResolved` will always be true and we are therefore just checking that `copy.canBeResolved` is true

I think the comment is less relevant if we are no longer comparing the copy's resolvable role to the original role.